portfolio/src/main/webapp/comments.js

```diff
@@ -1,8 +1,12 @@
 /* this file is a set of methods for fetching/creating comments */
 
+var authState = {};
```

Comment:
global variables are code smell. Generally, pass these global variables directly into methods that need it.